app/views/SecondContactCanWePhoneView.scala.html

```diff
@@ -32,15 +32,19 @@
             @govukErrorSummary(ErrorSummaryViewModel(form))
         }
 
+        <h1 class="govuk-heading-l">@messages("secondContactCanWePhone.heading", secondContactName)</h1>
```

Comment:
Minor change but we can use the view component for headings and paragraphs 😎 - 
```suggestion
  @heading(messages("secondContactCanWePhone.heading", secondContactName))
```